api/server/v1/tx.go

```diff
@@ -14,17 +14,56 @@
 
 package api
 
-func IsTxSupported(req Request) bool {
-	switch req.(type) {
-	case *InsertRequest, *ReplaceRequest, *UpdateRequest, *DeleteRequest, *ReadRequest,
-		*CreateOrUpdateCollectionRequest, *DropCollectionRequest, *ListCollectionsRequest:
+import (
+	"context"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/proto"
+)
+
+var (
+	HeaderPrefix = "Tigris-"
```

Comment:
should we add `header.go` in api/server/v1 then? I am thinking then we can also move the timeout header here as well?